### DISCUSSION TITLE
"installation type not supported for this application" when i try to add it to my server

### DISCUSSION THREAD

**Discussion by thecatgut** [jagrosh/MusicBot#1787] [Troubleshooting]
so when i try and add my jmusic but to my server i get "installation type not supported for this application" so i cant add it to my server but i cant turn on guild install to fix it because then i can have it be private

**Comment by malajvan** [comment]
had the same issue but i just turn on public mode, turn on guild, run the bot, add it to the server with the link (the bot will error out), turn off guild install and switch back to private mode. Works afterwards.

**Comment by vandevsysX64** [comment]
bot needs to be private or else have yt knocking at youre door
